types/util.d.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @return {void}
+ */
+export function bindMethods(): void;
+export class bindMethods {
+}
```

Comment:
are you sure this is needed? maybe line #4 alone is enough?

Reply:
Since you're binding methods to `this` it's considered a class constructor, so technically it's needed for TypeScript.